components/inputs/docs/input-checkbox.md

```diff
@@ -75,15 +76,10 @@ checkbox.addEventListener('change', (e) => {
 
 ### Slots
 
-* `default`: Primary text that will appear next to the input box and function as the label for the checkbox.
 * `inline-help`: Help text that will appear below the input. Use this only when other helpful cues are not sufficient, such as a carefully-worded label.
 * `supporting`: Supporting information which will appear below and be aligned with the checkbox.
 <!-- docs: end hidden content -->
 
-### Methods
-
-- `simulateClick()`: useful for testing, it simulates the user clicking on the checkbox, which toggles the state of the checkbox and fires the `change` event
```

Comment:
This shouldn't be necessary now that `clickElem` exists in test helpers.

Reply:
Should we spin up a story to clean up `simulateClick` usage so we can eventually remove it?

Reply:
I can -- there are 14 uses currently. Mostly I didn't want to snowball this even more, since cleaning those up is going to involve switching them to `@brightspace-ui/testing` if they're not already.